components/authz-service/storage/postgres/postgres.go

```diff
@@ -37,12 +37,16 @@ func New(ctx context.Context, migConf migration.Config, dataMigConf datamigratio
 	return db, nil
 }
 
-func initPostgresDB(ctx context.Context, pgURL string) (*sql.DB, error) {
+func initPostgresDB(ctx context.Context, pgURL string, maxConnections int32) (*sql.DB, error) {
 	d, err := db.PGOpenContext(ctx, pgURL)
 	if err != nil {
 		return nil, err
 	}
 
+	if maxConnections > 0 {
+		d.SetMaxOpenConns(maxConnections)
```

Comment:
I think you'll need to convert this to an int here or at some point in the call chain

```suggestion
		d.SetMaxOpenConns(int(maxConnections))
```

Reply:
we're setting it to `int` here: https://github.com/chef/automate/pull/3442/files#diff-7b11e67859dabc4eb026fda620cc2815R23